src/queries.js

```diff
@@ -0,0 +1,7 @@
+import CozyClient, { Q } from 'cozy-client'
+
+export const appsConn = {
+  query: Q('io.cozy.apps'),
+  as: 'apps',
```

Comment:
We have normalized the way we fetch one document by naming the query : `as: 'io.cozy.doctype/id'`. Should we do the same for queries fetching Q(doctype)? In order to have mutualisation somewhere? Something like `as: 'io.cozy.doctype/query'` ?

Reply:
I am not sure, it depends on the app also. I am not it's necessary to do that.